src/bsoncxx/oid.hpp

```diff
@@ -104,7 +107,10 @@ class BSONCXX_API oid {
     ///
     /// Conversion operator that indicates that the oid is initialized.
     ///
-    /// @return True if the oid is initialized.
+    /// @deprecated
```

Comment:
Done, and since this caught a use of the operator bool() in the JSON library I got rid of that too.